[2025-07-13T16:11:04.385+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-13T16:11:04.411+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T16:10:00+00:00 [queued]>
[2025-07-13T16:11:04.422+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T16:10:00+00:00 [queued]>
[2025-07-13T16:11:04.422+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-13T16:11:04.434+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): split_and_save_data> on 2025-07-13 16:10:00+00:00
[2025-07-13T16:11:04.443+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=799) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-13T16:11:04.445+0000] {standard_task_runner.py:72} INFO - Started process 802 to run task
[2025-07-13T16:11:04.446+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'employee_ingestion_dag', 'split_and_save_data', 'scheduled__2025-07-13T16:10:00+00:00', '--job-id', '384', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_pipeline.py', '--cfg-path', '/tmp/tmpkhhrjfww']
[2025-07-13T16:11:04.447+0000] {standard_task_runner.py:105} INFO - Job 384: Subtask split_and_save_data
[2025-07-13T16:11:04.506+0000] {task_command.py:467} INFO - Running <TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T16:10:00+00:00 [running]> on host a0a54bfbbb9e
[2025-07-13T16:11:04.595+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='employee_ingestion_dag' AIRFLOW_CTX_TASK_ID='split_and_save_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-13T16:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-13T16:10:00+00:00'
[2025-07-13T16:11:04.596+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-13T16:11:05.079+0000] {logging_mixin.py:190} INFO - Processing file: employee_data_with_errors_chunk_1.csv
[2025-07-13T16:11:05.079+0000] {logging_mixin.py:190} INFO - Total rows: 100, Invalid rows: 100
[2025-07-13T16:11:05.085+0000] {logging_mixin.py:190} INFO - File moved to bad_data: /opt/airflow/data/bad_data/employee_data_with_errors_chunk_1.csv
[2025-07-13T16:11:05.090+0000] {logging_mixin.py:190} INFO - Original file removed: /opt/airflow/data/raw_data/employee_data_with_errors_chunk_1.csv
[2025-07-13T16:11:05.091+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-13T16:11:05.103+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-13T16:11:05.103+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=employee_ingestion_dag, task_id=split_and_save_data, run_id=scheduled__2025-07-13T16:10:00+00:00, execution_date=20250713T161000, start_date=20250713T161104, end_date=20250713T161105
[2025-07-13T16:11:05.185+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-13T16:11:05.205+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-13T16:11:05.207+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
